bin/update_mirror.py

```diff
@@ -56,8 +56,16 @@ def lock(path):
 if socket.gethostname() == "repo-itb.opensciencegrid.org":
     hostname="repo-itb.opensciencegrid.org"
 
+def tagsplit(tag):
+    if 'upcoming' in tag:
```

Comment:
We can do it like this to be more explicit, if you like..?

```suggestion
    if 'upcoming' in tag and tag.startswith("osg-3."):
```

Reply:
yeah, please.